net/Directory.Build.props

```diff
@@ -7,6 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="NSubstitute.Analyzers.CSharp" >
```

Comment:
This is weird to reference Nsubstitute ananlyzer in every single project when it is required just for a single test one